crates/uv-scripts/src/lib.rs

```diff
@@ -163,11 +157,13 @@ fn extract_script_tag(contents: &[u8]) -> Result<Option<String>, Pep723Error> {
 
         // Otherwise, the line _must_ start with ` `.
         let Some(line) = line.strip_prefix(' ') else {
+            python_script.push(line);
+            python_script.extend(lines);
```

Comment:
Since we break, doesn't the `python_script` get truncated?

Reply:
`python_script` gets all of the remaining lines for the files before we break so it should contain the rest of the script file.